examples/builder_derive_interop/4_flatten_hand_args.md

```diff
@@ -0,0 +1,7 @@
+*[Jump to source](4_flatten_hand_args.rs)*
+
+# 4: Flatten hand-implemented args into derived CLI
```

Comment:
We should have fenced code blocks (type `console`) to test these with `trycmd`.

If they don't seem to fit within the story of the text, we could have them in a separate file.